distance-frontend/src/components/toolbox/Dropdown.jsx

```diff
@@ -16,7 +16,7 @@ const Dropdown = (props) => {
         {options.map((o) => (
           <option
             key={o.label}
-            selected={value === o.label}
```

Comment:
i changed this back to selected last time because it wasn't populating the default value. we need to make sure this works.